### DISCUSSION TITLE
[Vote ended on 2023-03-30] Adopting Nikola as the static site generator for the first iteration of the Ansible community web presence

### DISCUSSION THREAD

**Discussion by oraNod** [ansible-community/community-topics#216] [Votes done]
### Summary

This proposal is to adopt Nikola (https://getnikola.com/) as the static site generator to build and deploy the Ansible community website. After discussion and careful consideration the Ansible community team suggests that Nikola is the best choice for the initial launch and early development of the Ansible community website. 

This vote does not result in a permanent decision regarding the tooling. The Ansible community should continually assess needs and evaluate options. If we determine at a future stage that another SSG offers more options or better functionality then we should be able to replace the underlying SSG without much difficulty.

Please vote with +1 / -1 to indicate preference for using Nikola.

### Additional Information

Please refer to this topic for more discussion as well as details about the static site generators brought forward for consideration from the community: https://github.com/ansible-community/community-topics/issues/210

**Comment by cybette** [comment]
+1

**Comment by anweshadas** [comment]
+1

**Comment by gotmax23** [comment]
This vote was not correctly created. The +`1`s and `-1`s need to be in a Github Discussion, and the other discussion should be in a separate issue.

**Reply by gotmax23** [reply]
I've fixed it. I'll formally vote `+1` after I've played around a little more with Nikola.

**Reply by oraNod** [reply]
Thank you for fixing it @gotmax23 apologies for the confusion.

**Reply by gotmax23** [reply]
No problem!

**Comment by felixfontein** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by samccann** [comment]
+1

**Comment by leogallego** [comment]
+1

**Comment by apple4ever** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by GregSutcliffe** [comment]
+1

**Comment by maxamillion** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by tonk** [comment]
+1
